packages/st2/Makefile

```diff
@@ -30,6 +30,10 @@ else
 	DEB_DISTRO := $(shell lsb_release -cs)
 endif
 
+# Makefile function to retry the failed command 'N' times
+# Example: $(call retry,3,some_script.sh)
+retry = $(2) $(foreach t,$(shell seq 1 ${1}),|| (echo -e "\033[33m Failed ($$?): '$(2)'\n Retrying $t ... \033[0m"; $(2)))
```

Comment:
Yeah, I don't like it too. But that's how it's usually done in make: https://www.gnu.org/software/make/manual/make.html#Call-Function

Multi-line would be even worse because of strange formatting stuff like putting `\` in the end of every line which is more error prone and then calling that function making sure positional arguments weren't messed.

I'll play with retries builds https://circleci.com/gh/StackStorm/st2-packages/tree/feature%2Fretry-on-failure more and if I'll find that `1` single command retry was enough in all cases,
maybe just:
```sh
pip wheel --wheel-dir=$(WHEELDIR) --find-links=$(WHEELDIR) -r requirements.txt || \
  pip wheel --wheel-dir=$(WHEELDIR) --find-links=$(WHEELDIR) -r requirements.txt
```

to remove function with echo-s and that makefile alien syntax to avoid entire complexity.